apps/authorization/tests/test_data_access_grant.py

```diff
@@ -169,72 +183,74 @@ def test_thirteen_month_app_type_with_switch_limit_data_access(self):
-            'client_id': application.client_id,
-            'response_type': 'code',
-            'redirect_uri': redirect_uri,
-        }
-        response = self.client.get('/v1/o/authorize', data=payload)
-        # post the authorization form with only one scope selected
-        payload = {
-            'client_id': application.client_id,
-            'response_type': 'code',
-            'redirect_uri': redirect_uri,
-            'scope': ['capability-a'],
-            'expires_in': 86400,
-            'allow': True,
-        }
-        response = self.client.post(response['Location'], data=payload)
+    @override_switch('expire_grant_endpoint', active=True)
+    def test_delete_authenticated_user_grant(self):
+        # 1. create user, app, ac
+        user, application, ac = self._create_user_app_token_grant(
+            first_name="first",
+            last_name="last1",
+            fhir_id="-20140000008325",
+            app_name="test_app1",
+            app_username="devuser1",
+            app_user_organization="org1",
+        )
 
-        self.assertEqual(response.status_code, 302)
-        # now extract the authorization code and use it to request an access_token
-        query_dict = parse_qs(urlparse(response['Location']).query)
-        authorization_code = query_dict.pop('code')
-        token_request_data = {
-            'grant_type': 'authorization_code',
-            'code': authorization_code,
-            'redirect_uri': redirect_uri,
-            'client_id': application.client_id,
-        }
-        response = self.client.post(reverse('oauth2_provider:token'), data=token_request_data)
-        self.assertEqual(response.status_code, 200)
+        user, application_2, ac = self._create_user_app_token_grant(
+            first_name="first",
+            last_name="last1",
+            fhir_id="-20140000008325",
+            app_name="test_app2",
+            app_username="devuser2",
+            app_user_organization="org2",
+        )
 
-        # errors if DNE or more than one is found
+        self.client.logout()
+        # 2. verify grant creation - errors if DNE or more than one is found
         DataAccessGrant.objects.get(beneficiary=user.id, application=application.id)
+        DataAccessGrant.objects.get(beneficiary=user.id, application=application_2.id)
 
-    def test_no_action_on_reapproval(self):
+        # 3. logout
+        self.client.logout()
+
+        # 4. call without authentication and expect 403
+        response = self.client.post('/v1/o/expire_authenticated_user/{0}/'.format("-20140000008325"))
+        self.assertEqual(response.status_code, 403)
+
+        # 5. Create authentication headers and expect success
+        auth = self._create_authorization_header(application.client_id, application.client_secret)
+        self.client.post('/v1/o/expire_authenticated_user/{0}/'.format("-20140000008325"),
+                         HTTP_AUTHORIZATION=auth,
+                         )
+
```

Comment:
```suggestion
self.assertEqual(response.status_code, 200)
```
Should check the response status is successful here.

In testing, I'm getting the following:

```
AssertionError: 403 != 200
```